packages/peregrine/lib/talons/Header/useAccountTrigger.js

```diff
@@ -19,9 +19,13 @@ import { clearCustomerDataFromCache } from '@magento/peregrine/lib/Apollo/clearC
  * @returns {Boolean}   talonProps.isUserSignedIn - Whether there is a user currently signed in or not.
  */
 export const useAccountTrigger = props => {
-    const { mutations } = props;
+    const { mutations, VIEWS } = props;
     const { signOut: signOutMutation } = mutations;
 
+    const [formErrors, setFormErrors] = useState([]);
+    const [view, setView] = useState(VIEWS.SIGNIN);
```

Comment:
I don't think account trigger needs to be the owner of the view state - that can belong in the `AccountMenu` component.

Reply:
This still feels odd to me, what do you think about moving ownership of the view to AccountMenu?

Reply:
I agree with you on that. Moved the logic to the new account menu talon. 4e21c97